x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/TrainedModelConfig.java

```diff
@@ -182,15 +171,15 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         if (description != null) {
             builder.field(DESCRIPTION.getPreferredName(), description);
         }
-        builder.timeField(CREATED_TIME.getPreferredName(), CREATED_TIME.getPreferredName() + "_string", createdTime.toEpochMilli());
-        builder.field(MODEL_VERSION.getPreferredName(), modelVersion);
-        builder.field(MODEL_TYPE.getPreferredName(), modelType);
+        builder.timeField(CREATE_TIME.getPreferredName(), CREATE_TIME.getPreferredName() + "_string", createTime.toEpochMilli());
         if (definition != null) {
             builder.field(DEFINITION.getPreferredName(), definition);
         }
+        builder.field(TAGS.getPreferredName(), tags);
         if (metadata != null) {
             builder.field(METADATA.getPreferredName(), metadata);
         }
+        builder.field(InferenceIndexConstants.DOC_TYPE.getPreferredName(), NAME);
```

Comment:
I think it is wise to only marshall this to XContent if it is being stored. Not a huge deal, but I doubt users will need to see this with a GET call. 

If pushed for time, we can add the condition later.